app/controllers/profile_controller.rb

```diff
@@ -15,20 +9,29 @@ def show
   end
 
   def edit
-    @profile = User.find(session[:id])
-    @id = session[:id]
+    @profile = User.find(params[:id])
```

Comment:
Just curious why are we moving away from session here?

Reply:
Originally, this feature was merged before the session omniauth feature was merged. So it's still the code from that time.